RollaProject/kernel
# feat: pull and merge changes from ZeroDev's upstream v2.1

- Fix/stake factory (#7)
- Feat/multi owned ecdsa (#8)
- ERC721 actions renamed to token actions and added erc20, erc1155 (#10)
- Feat/killswitch (#11)
- udpated the killswitch action (#12)
- fix: return value on temp kernel (#13)
- deploy script
- Update audit report to include session keys
- deployed to optimism, polygon, arbitrum
- V2.1 (#25)
- Add audit report
- Update audit report to latest version
- V2.1 typed primitives (#26)
- Deploy/v2.1 (#28)
- Deploy/v2.1 (#29)
- added stake functionality to factory
- added stake factory to follow the storage access rule for associated storage
- renamed to TempKernel
- deploy script
- changed script a bit
- renamed to TempKernel
- forge fmt
- removed console.sol
- removed staker and renamed actions to executor
- deployed with deterministic deployment
- script done for session key
- using simple trick to adapt the storage access rule
- feat: added a new SessionKeyOwnedKernelFactory
- fix: SessionKeyOwnedKernel owner data for account creation
- refactor: expect digest passed to be a hashed typed data already
- Revert "refactor: expect digest passed to be a hashed typed data already"
- refactor: validate signed typed data and signed eth messages
- feat: pull and merge changes from ZeroDev's upstream v2.1
